pppd/eap-tls.c

```diff
@@ -305,25 +312,30 @@ SSL_CTX *eaptls_init_ssl(int init_server, char *cacertfile, char *capath,
     long         tls_version = TLS1_VERSION; 
 #endif
 
+    info("Using PKCS12: %s", pkcs12);
```

Comment:
Do we really want to print this if no pkcs12 file was specified? It would be confusing to the user to print "Using PCKS12" when we aren't.

Reply:
This is already fixed in a later change